### ISSUE TITLE
better error messages

### ISSUE THREAD

**Issue by rabbah** [apache/incubator-openwhisk-wskdeploy#925]
run `wskdeploy` with no manifest file and you get this

```
Error: root.go [140]: [ERROR_MANIFEST_FILE_NOT_FOUND]: File: [openwhisk_wskdeploy-latest-mac-386]: Manifest file not found at path [/Users/rabbah/Downloads/openwhisk_wskdeploy-latest-mac-386].
```

I was expecting a "usage" method/help. In any case, I suggest a better formatted error message that eschews all the internal details... perhaps if no file is found, suggest `wskdeploy -h` or just print usage info with a message that no manifest was found.

**Comment by rabbah** [comment]
many error messages could be reported better fwiw.

**Comment by pritidesai** [comment]
@rabbah, @houshengbo addressed this in his PR #943 

![image](https://user-images.githubusercontent.com/206285/40738135-95fc9038-63f7-11e8-803c-07f921462b03.png)

**Comment by pritidesai** [comment]
Closing it as its fixed now.
